build.gradle

```diff
@@ -58,9 +57,12 @@ versioneye {
 }
 
 allprojects {
+    apply plugin: 'java'
     apply plugin: 'jacoco'
     apply plugin: 'nebula.lint'
 
+    sourceCompatibility = 1.8
```

Comment:
This item looks like something that we can move into `gradle.properties` (and that would be a better fit there).